### DISCUSSION TITLE
How to pass environment file path to export variables from it?

### DISCUSSION THREAD

**Discussion by linkuha** [mlflow/mlflow#8140] [Q&A]
supposably we can pass only a file with secrets from Vault (in k8s), how we can achieve read vars from it? plan you extend arguments (e.g. --env-path) for support this case?

**Comment by JHibbard** [comment]
Hi Linkuha,

I think this sounds more like a K8s question or maybe a feature request. Could you open up a feature request in MLflow's GitHub repo and link to this question? I think that's the best way to get your question answered.

Hope that helps!
Jim
